tests/staking/PauseStaking.test.js

```diff
@@ -351,7 +355,7 @@ contract("Staking", (accounts) => {
             });
         });
 
-        it("should not allow governanceWithdrawTokens when frozen", async () => {
+        it("should not allow other than team vesting to withdraw from cancelTeamVesting", async () => {
```

Comment:
pls move test to a relevatn test file - it has nothing to do with PauseStaking tests